src/applications/disability-benefits/781/containers/App.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import RoutedSavableApp from '../../../../platform/forms/save-in-progress/RoutedSavableApp';
+import formConfig from '../config/form';
+
+export default function App({ location, children }) {
```

Comment:
Instances of `App` along with their filenames, are typically qualified, such as `Form781App`.

Reply:
Will do!